docs/user/user-defined-redirects.rst

```diff
@@ -1,39 +1,116 @@
+`Built-in redirects`_ ⬇️
+    Allows for simple and long-term sharing of external references to your documentation.
 
-Your redirects will be effective immediately.
+`User-defined redirects`_ ⬇️
+    Makes it easier to move contents around (see: )
 
-Features
---------
+.. seealso::
+
+   :doc:`/guides/redirects`
+     This guide shows you how to add redirects with practical examples.
+   :doc:`/automatic-redirects`
+     Information and tips about creating and handling external references.
+   :doc:`/guides/deprecating-content`
+     A guide to deprecating features and other topics in a documentation.
+
+
+Special features
+----------------
 
 - By default, redirects are followed only if the requested page doesn't exist
-  (*404 File Not Found* error), if you need to apply a redirect for files that exist,
-  mark the :guilabel:`Force redirect` option.
+  (*404 File Not Found* error).
+  If you need to apply a redirect for files that exist,
   **This option is only available on some plan levels**.
   Please ask support if you need it for some reason.
 - :ref:`user-defined-redirects:page redirects` and :ref:`user-defined-redirects:exact redirects`
   can redirect to URLs outside Read the Docs,
   just include the protocol in ``To URL``, e.g ``https://example.com``.
 
-Redirect types
---------------
 
-We offer a few different type of redirects based on what you want to do.
+Built-in redirects
+------------------
+
+This article explains the redirects that are automatically active for all Read the Docs projects and how they are useful.
+Automatic redirects on Read the Docs are supported long-term.
+
+Built-in redirects are especially useful for creating and sharing **incoming links**
+which is discussed indepth in :doc:`/automatic-redirects`.
+
+Page redirects at ``/page/``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can link to a specific page and have it redirect to your default version,
+allowing you to create links on external sources that are always up to date.
+This is done with the ``/page/`` URL prefix.
+
+For instance, you can reach the page you are reading now by going to https://docs.readthedocs.io/page/automatic-redirects.html.
+
+Another way to handle this is the ``latest`` version.
+You can set your ``latest`` version to a specific version and just always link to ``latest``.
+You can reach this page by going to https://docs.readthedocs.io/en/latest/automatic-redirects.html.
+
+
+Root URL redirect at ``/``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A link to the root of your documentation (`<slug>.readthedocs.io/`) will redirect to the  :term:`default version`,
+as set in your project settings.
+
+This works for both readthedocs.io (|org_brand|), readthedocs-hosted.com (|com_brand|), and :doc:`custom domains </custom-domains>`.
+
+For example::
+
+    docs.readthedocs.io -> docs.readthedocs.io/en/latest/
+    www.pip-installer.org -> www.pip-installer.org/en/latest/
+
+.. warning::
+
+   This only works for the root URL, not for internal pages.
```

Comment:
What are internal pages?

Reply:
I don't know + this whole warning box is weird. Look at the warning text below.

Gonna delete it...

Reply:
I think we should combine the two warnings to a single `note::` that says

> You cannot use the root redirect to reference specific pages. `/` redirects to the default version, but `/some/page.html` will not redirect to `/en/latest/some/page.html`. Use the `/page/` redirect rule for this purpose.

Reply:
Made it a warning box.